src/main/java/com/websafe/ccmsbe/controller/WebsiteController.java

```diff
@@ -54,4 +54,9 @@ public Website updateWebsite(
         return  websiteService.updateWebsite(userId,website);
     }
 
+    @GetMapping("/getPR/{websiteId}")
```

Comment:
This should be something like `api/vi/websites/{websiteId}/privacy-regulations`. Shall we refactor the code accordingly. 

```suggestion
    @GetMapping("/websites/{websiteId}/privacy-regulations")
```

Reply:
API Endpoint Naming: The naming convention for API endpoints could be improved. Specifically, operation names (create, get, update, delete, etc.) should not be used in the endpoint path. It is suggested to use a path such as api/vi/websites/{websiteId}/privacy-regulations instead